.github/workflows/sync-docs-from-node.yml

```
@@ -26,878 +26,342 @@ on:
-          echo "" >> $SYNC_REPORT
+  aggregate-results:
+    name: 'Aggregate Sync Results'
+    runs-on: ubuntu-latest
+    needs: [prepare, sync-files]
+    if: always() && needs.prepare.outputs.should_continue == 'true'
+    outputs:
+      total_changes: ${{ steps.calculate.outputs.total_changes }}
+      total_added: ${{ steps.calculate.outputs.total_added }}
+      total_updated: ${{ steps.calculate.outputs.total_updated }}
+      total_deleted: ${{ steps.calculate.outputs.total_deleted }}
+      sync_reports: ${{ steps.collect.outputs.all_reports }}
```

actionlint (1.7.7)

property "collect" is not defined in object type {calculate: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)